ldb/models.py

```diff
@@ -182,9 +238,11 @@ def age(self):
             return today.year - born.year
 
     def save(self):
+        self._membership_status = self.membership_status
```

Comment:
Meh... een deel van me zegt dat we dat wel moeten doen, maar eigenlijk is het onzin en moeten we (ik) er maar mee leren leven dat het in de save zit :wink: